docs/conf.py

```diff
@@ -14,17 +14,17 @@
 import sys
 
 
-sys.path.insert(0, os.path.abspath('../'))
+sys.path.insert(0, os.path.abspath("../"))
```

Comment:
Why is this needed?

Reply:
I did not change the code, black simply changed the quotations marks used. I don't think it should be needed. I was just linting and formatting.